theme/theme.go

```diff
@@ -545,6 +545,36 @@ func loadCustomFont(env, variant string, fallback fyne.Resource) fyne.Resource {
 	return res
 }
 
+func safeColorLookup(n fyne.ThemeColorName, v fyne.ThemeVariant) color.Color {
+	col := current().Color(n, v)
+	if col == nil {
+		return &color.NRGBA{}
+	}
+	return col
+}
+
+func safeFontLookup(s fyne.TextStyle) fyne.Resource {
+	col := current().Font(s)
+	if col != nil {
+		return col
+	}
+
+	if s.Monospace {
+		return DefaultTextMonospaceFont()
+	}
+	if s.Bold {
+		if s.Italic {
+			return DefaultTextBoldItalicFont()
+		}
+		return DefaultTextBoldFont()
+	}
```

Comment:
I’d prefer
```go
if s.Bold && s.Italic {
	return DefaultTextBoldItalicFont()
}
if s.Bold {
	return DefaultTextBoldFont()
}
```
But I’ll leave the decision to you.

Reply:
I was matching the way it's done in all the other font style comparisons